extensions/vault/runtime/src/main/java/io/quarkus/vault/VaultDynamicDbSecretService.java

```diff
@@ -0,0 +1,96 @@
+package io.quarkus.vault;
+
+import javax.ws.rs.WebApplicationException;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.vault.proxy.dto.VaultCreds;
+import io.quarkus.vault.proxy.dto.VaultLeasesBody;
+import io.quarkus.vault.proxy.dto.VaultRenewLease;
+import io.quarkus.vault.service.dto.LeaseBase;
+import io.quarkus.vault.service.dto.VaultDynamicCredentials;
+
+/**
+ * handles db dynamic credentials. supports revocation and renewals.
+ */
+public class VaultDynamicDbSecretService {
```

Comment:
Should it be called just `VaultDynamicSecretService` ? (Since I hope all of the Vault can be used not only for `agroal`)

Reply:
this class refers specifically to https://www.vaultproject.io/docs/secrets/databases/index.html, which is targeting databases only.

Reply:
I see, thanks